doc/components_1.md

```diff
@@ -27,16 +27,78 @@ For instance, the `mj-button` component is, on the inside, a complex HTML layout
 </table>
 ```
 
+#### Attribute lists
```

Comment:
I have still mixed feelings about this whole section, here you describe how a table works and as reading this i'm feeling a bit infantilized.

Reply:
"infantilized": Great word, but never in a million years would I have come up with it.

On the other hand, I feel infantilized about the mismatch of numbers here to my screen. I "pulled next" and everything got clearer. 😔

I see your point. I shortened it a lot. Good feedback.